### PR TITLE
1 Project Setup and Creating the Business Model

### PR DESCRIPTION
**Pull request by leonardk1**
#1
As A Developer
I Want To Create the Business Model
So That it shall be used in implementing features related with the Business

#### NB
- the large number of files is actually due to `$ binit` mainly, but basically the PR is not big

#### tasks
- [x] create the business model
- [x] configure the test environment

### CODE REVIEW

**Comment by leonardk1** [comment]
@godfreym please review

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,7 @@
 !/log/.keep
 /tmp
 vendor/ruby
+
```
Comment (godfreym) [inline .gitignore]: @leonardk1 no need of extra white spaces

**Comment by godfreym** [comment]
@leonardk1 The PR title does not reflect the contents, it is the project setup pr + business model

**Comment by godfreym** [comment]
@leonardk1 Apart from my comments, the code looks fine :+1:

**Comment by leonardk1** [comment]
@godfreym corrected
@arafatm please review

**Comment by godfreym** [comment]
@leonardk1 :+1:

**Comment by leonardk1** [comment]
@arafatm please review

**Comment by arafatm** [comment]
@leonardk1 :+1:

**Pull request merged**